content/en/docs/concepts/services-networking/ingress.md

```diff
@@ -219,23 +219,48 @@ of the controller that should implement the class.
 
 {{< codenew file="service/networking/external-lb.yaml" >}}
 
-IngressClass resources contain an optional parameters field. This can be used to
-reference additional implementation-specific configuration for this class.
+The `.spec.parameters` field of an IngressClass lets you reference another
+resource that provides configuration related to that IngressClass.
+
+The specific type of parameters to use depends on the ingress controller
```

Comment:
Should Ingress be capitalized here?

Reply:
I usually favor “ACME Net is an open source ingress controller” but would write  “create a new IngressClass” or “the beta Ingress API is deprecated, get upgrading”.